dashboard/src/components/Config/AppRepoList/AppRepoControl.test.tsx

```diff
@@ -67,7 +68,8 @@ it("deletes the repo and refreshes list", async () => {
   expect(fetchRepos).toHaveBeenCalledWith(defaultProps.namespace, defaultProps.kubeappsNamespace);
 });
 
-it("deletes the repo and refreshes list (in other namespace)", async () => {
+// TODO: Re-enable when the repo view is refactored
+xit("deletes the repo and refreshes list (in other namespace)", async () => {
```

Comment:
In retrospect, would it have been an option to update the AppList component without disabling the "All Namespaces" functionality (so other components continued to work) until you'd finished? Otherwise you may need to hold off landing this PR until the others are ready (not a big issue either).

Reply:
well, yes, it would've been in a inconsistent state anyway so I prefer to use this temporary branch since there are not that many changes more.